### PR TITLE
Add EIP: MetaNFT keystore contracts

### PR DESCRIPTION
**Pull request by galimba**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-7210.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+eip: 7210
```
Comment (abcoathup) [inline EIPS/eip-7210.md:2]:
```suggestion
eip: 7208
```
Manually assigning EIP number 7208.  Please also rename the files.

This PR was created 35 minutes after #7207 and #7208 and #7209 are missing.  EIP number sniping and gaming is not allowed

Comment (galimba) [reply EIPS/eip-7210.md:2]: oh, man hhahaha... 7210 would have been the perfect number for this standardization :sweat_smile: Thanks for reviewing the PR so quickly.

What's most important is the interface, looking forward to discussing how to standardize it.

**Comment by galimba** [comment]
any updates on this PR?

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Comment by SamWilsn** [comment]
@galimba sorry it's taken so long for someone to look at your PR! I think I'm caught up to about June...

**Comment by galimba** [comment]
Thanks for the update @SamWilsn .
This ERC is under active development, we'll make sure to follow the guide and migrate accordingly.

**Pull request closed without merging**